utils/releaser/src/ReleaseWorker/ReleaseCandidate/GenerateApiaryBlueprintReleaseWorker.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\Releaser\ReleaseWorker\ReleaseCandidate;
+
+use PharIo\Version\Version;
+use Shopsys\Releaser\ReleaseWorker\AbstractShopsysReleaseWorker;
+use Shopsys\Releaser\Stage;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+final class GenerateApiaryBlueprintReleaseWorker extends AbstractShopsysReleaseWorker
+{
+    /**
+     * @param \Symfony\Component\Console\Style\SymfonyStyle $symfonyStyle
+     */
+    public function __construct(SymfonyStyle $symfonyStyle)
+    {
+        $this->symfonyStyle = $symfonyStyle;
+    }
+
+    /**
+     * @param \PharIo\Version\Version $version
+     * @return string
+     */
+    public function getDescription(Version $version): string
+    {
+        return 'Generate Apiary.io blueprint "php phing frontend-api-generate-apiary-blueprint" and commit it';
+    }
+
+    /**
+     * Higher first
+     *
+     * @return int
+     */
+    public function getPriority(): int
+    {
+        return 870;
+    }
+
+    /**
+     * @param \PharIo\Version\Version $version
+     */
+    public function work(Version $version): void
+    {
+        $this->processRunner->run('php phing frontend-api-generate-apiary-blueprint');
+
+        if ($this->hasGeneratedBlueprint()) {
+            $this->commit('generate apiary.io blueprint');
```

Comment:
_nitpick_

I would use past tense so it's in the same style as other commits during release (`version updated ...`, `locked versions ...`, `Changelog was updated`, ... )

Reply:
changed